certified-connectors/OrbusInfinity/apiProperties.json

```diff
@@ -0,0 +1,70 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oAuthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "scopes": [
+            "user_impersonation"
+          ],
+          "redirectMode": "GlobalPerConnector",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "{resourceUri}",
+            "IsOnbehalfofLoginSupported": true
+          },
+          "customParameters": {
+            "LoginUri": {
+              "value": "https://login.microsoftonline.com"
+            },
+            "TenantId": {
+              "value": "common"
+            },
+            "EnableOnbehalfOfLogin": {
+              "value": false
+            }
+          },
+          "clientId": "<<Please add your clientID here>>"
+        },
+        "uiDefinition": null
+      },
+      "token:tenantId": {
+        "type": "string",
```

Comment:
Please fix the below error, kindly refer the schema here for help. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json

##[error]@{level=Error; message=Property 'metadata' has not been defined and the schema does not allow additional properties.; valuePath=line: 34 position: 19 path: '[0].properties.connectionParameters.token:tenantId.metadata'; value=metadata}

Reply:
fixed